chall11/gsanchez.c

```diff
@@ -0,0 +1,77 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   gsanchez.c                                         :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: gsanchez <gsanchez@student.42madrid.com    +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2020/05/21 23:20:53 by gsanchez          #+#    #+#             */
+/*   Updated: 2020/05/22 11:32:12 by gsanchez         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <ctype.h>
+
+char  **ft_parse_nlarray(char *src, int newsize)
+{
+  char  **arr;
+  int   j;
+
+  arr = malloc(sizeof(char*)*newsize);
+  j = 0;
+  while (j != newsize)
+      arr[j++] = strtok_r(src, "\n", &src);
+  return (arr);
+}
+
+char  *ft_goto_parser(const char *code)
+{
+    char *buf, *result, *aux, *rest, *token;
+    char **arr;
+    int i, j, indextogo;
```

Comment:
You use snake case for your functions so I would use snake case for `indextogo` too. So -> `index_to_go`, more readable IMO.